tests/mixed-source/ios/MixedSourceFramework/MixedSourceFramework.h

```diff
@@ -1,12 +1,2 @@
-
-#import <Foundation/Foundation.h>
-#import <MixedSourceFramework/DoubleQuoteLogger.h>
-#import <MixedSourceFramework/DoubleQuoteNamespacedLogger.h>
-
-//! Project version number for MixedSourceFramework.
-FOUNDATION_EXPORT double MixedSourceFrameworkVersionNumber;
-
-//! Project version string for MixedSourceFramework.
-FOUNDATION_EXPORT const unsigned char MixedSourceFrameworkVersionString[];
-
-// In this header, you should import all the public headers of your framework using statements like #import <MixedSourceFramework/PublicHeader.h>
+// Needed because the generated -Swift.h header is hard-coded to import this file, instead of the underlying headers / umbrella header for the module
+#import <MixedSourceFramework/MixedSourceFramework-umbrella.h>
```

Comment:
there are cases when `SomeFramework.h` file has a list of includes, shouldn't we have a test case for this situation?

Reply:
Probably not needed, since I cant think of a functional difference?

Reply:
if no functional difference why need to change here then?

Reply:
easier than manually adding each header when a new one gets added